src/midend/binaryAnalyses/instructionSemantics/DataFlowSemantics2.C

```diff
@@ -77,6 +77,7 @@ public:
     createOptionalMerge(const BaseSemantics::SValuePtr &other, const BaseSemantics::MergerPtr&,
                         SMTSolver*) const ROSE_OVERRIDE {
         TODO("[Robb P. Matzke 2015-08-10]");
```

Comment:
Great! It needs to go into src/util/Sawyer/Sawyer.h because the ASSERT_*
and TODO macros are part of Sawyer. Therefore, it should be named
SAWYER_NORETURN instead of ROSE_NORETURN, although a ROSE_NORETURN could
also be defined (perhaps next to wherever ROSE_OVERRIDE is defined). If
you send me the diff for the Sawyer changes I'll add them upstream.

On 11/25/2015 02:01 PM, Bill Hoffman wrote:
> In src/midend/binaryAnalyses/instructionSemantics/DataFlowSemantics2.C
> <https://github.com/rose-compiler/rose-develop/pull/5#discussion_r45906262>:
> 
>> @@ -77,6 +77,7 @@ public:
>>      createOptionalMerge(const BaseSemantics::SValuePtr &other, const BaseSemantics::MergerPtr&,
>>                          SMTSolver*) const ROSE_OVERRIDE {
>>          TODO("[Robb P. Matzke 2015-08-10]");
> 
> OK, this seems to compile:
> virtual __declspec(noreturn) Sawyer::OptionalBaseSemantics::SValuePtr
> So, it should be made into a macro. ROSE_NORETURN? Where should that
> macro go?
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rose-compiler/rose-develop/pull/5/files#r45906262>.
>